packages/amplication-server/src/core/block/block.util.ts

```diff
@@ -40,3 +42,35 @@ export const mergeAllSettings = (
     ...mergedObj,
   };
 };
+
+export async function validateEntitlement(
```

Comment:
let's rename to `validateBlockEntitlement` or `validateEntitlementForBlockType` so it's more clear